hack/test

```diff
@@ -49,8 +49,16 @@ function e2e_pass {
 }
 
 function unittest_pass {
+	echo "mode: atomic" > coverage.txt
```

Comment:
This may not be strictly necessary. It's just how `go test -covermode=<mode>` normally generates a coverage report file with the mode at the top.